### PR TITLE
Do not merge: try to cache eclipse.tar.gz

### PR DESCRIPTION
**Pull request by KengoTODA**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,10 +15,11 @@ matrix:
        apt:
          packages:
            - oracle-java9-installer
-     
+
 install:
-  - wget http://www.eclipse.org/downloads/download.php?file=/eclipse/downloads/drops/R-3.6.2-201102101200/eclipse-SDK-3.6.2-linux-gtk-x86_64.tar.gz -O eclipse-3.6.2-linux64.tar.gz
-  - tar xzvf eclipse-3.6.2-linux64.tar.gz eclipse
+  - if [ -e eclipse.tar.gz ]; then wget http://www.eclipse.org/downloads/download.php?file=/eclipse/downloads/drops/R-3.6.2-201102101200/eclipse-SDK-3.6.2-linux-gtk-x86_64.tar.gz -O eclipse.tar.gz; fi
```
Comment (jsotuyod) [inline .travis.yml]: This should be `if [[ ! -e eclipse.tar.gz ]]; then ...`

**Comment by jsotuyod** [comment]
Really interesting. This should help a lot with Travis flakyness!

**Comment by henrik242** [comment]
Haha, I didn't notice that you had also been working on the @KengoTODA (ref #71 )

**Comment by KengoTODA** [comment]
duplicated with #71

**Pull request closed without merging**